SimCalorimetry/EcalSimProducers/python/esEcalLiteDTUPedestalsProducer_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+EcalLiteDTUPedestalsRcd =  cms.ESSource("EmptyESSource",
+                                recordName = cms.string("EcalLiteDTUPedestalsRcd"),
+                                firstValid = cms.vuint32(1),
+                                iovIsRunNotTime = cms.bool(True)
+                                )
+
+EcalLiteDTUPedestals = cms.ESProducer("EcalLiteDTUPedestalsESProducer"),
```

Comment:
remove trailing comma

Reply:
not done

Reply:
because if i remove the trailing comma i get the error:
----- Begin Fatal Exception 21-Jul-2020 16:38:24 CEST-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing ESProducer: class=EcalLiteDTUPedestalsESProducer label='EcalLiteDTUPedestals'
Exception Message:
MissingParameter: Parameter 'ComponentName' not found.

Reply:
a trailing comma in python turns an object into a tuple. therefore, with the trailing comma, this is not actually treated as an ESProducer in the process. without the trailing comma, it is correctly treated as an incorrectly configured ESProducer, i.e. you need to specify the necessary parameters when you declare the ESProducer.

Reply:
aaah, gotcha, now i understand the meaning of those files. I fix it.